src/components/GeneralStats/styles.css

```
@@ -0,0 +1,60 @@
+.frame {
+  padding: 24.38px;
+  background-color: whitesmoke;
+}
+
+.statistics {
+  padding: 25px;
+  width: 100%;
+
+  border-radius: 8px;
+  background-color: #ffffff;
+  box-shadow: 0px 12px 26px rgba(16, 30, 115, 0.06);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.icon {
+  width: 59.68px;
```

```suggestion
  width: 60px;
```
Ponovo isto: najbolje je da zakoruzimo!